docs/release_notes/release_notes-0.15.0.rst

```diff
@@ -90,4 +90,4 @@ Contributors
 @jrsacher
 @orcahmlee
 @naren-ponder
-@RehanSD
+@RehanSD
```

Comment:
Can we remove this change from this PR?

Reply:
@vnlitvinov, please remove this change.

Reply:
TBH I have no idea why it isn't rolled back, I have tried it several times to revert to no avail for now.
Is this a blocker?..

Reply:
I think I had finally managed to beat it!